Update README file organizing sections and contents (#1122)
ChildMindInstitute/mindlogger-backend-refactor: README.md

# ChildMindInstitute - Mindlogger Backend API

This repository is used as a backend for the service MindLogger

## 1. Getting Started / Installation

### 1.1 Create `.env` file for future needs

> It is hightly recommended to create `.env` file as far as it is needed for setting up the project with Local and Docker approaches.

Reviewer: ```suggestion
It is highly recommended to create an `.env` file as far as it is needed for setting up the project with Local and Docker approaches.
```

Author: done

<br/>

## 2. Usage

### 2.1 Running locally

> 🛑 **NOTE:** Don't forget to set the `PYTHONPATH` environment variable, e.g: export PYTHONPATH=src/

In project we use simplified version of imports: `from apps.application_name import class_name, function_name, module_nanme`.

> For doing this we must have `src/` folder specified in a **PATH**.

Reviewer: ```suggestion
To do this this we must have `src/` folder specified in a **PATH**.
```

Author: done
